### PR TITLE
Add Speakers section to homepage

### LINKED ISSUE #56: Prepare keynote speakers section block on landing page
We're going to add the keynote speakers soon. 
Keynote speaker section block needs to be added on [this](generate_site/templates/index-base.html)
Design on desktop is shown [here](https://www.figma.com/file/ow9vzSa50YsUDaI65RJB81/PyLadies-Conference?type=design&node-id=123-2674&mode=design&t=WK0eMN0gnCmrKozn-4), on tablet [here](https://www.figma.com/file/ow9vzSa50YsUDaI65RJB81/PyLadies-Conference?type=design&node-id=123-1814&mode=design&t=WK0eMN0gnCmrKozn-4) and on mobile [here](https://www.figma.com/file/ow9vzSa50YsUDaI65RJB81/PyLadies-Conference?type=design&node-id=141-4896&mode=design&t=WK0eMN0gnCmrKozn-4) 
- to hide the block first and the team will unhide once all keynote speakers are ready.

### PR DESCRIPTION
**Pull request by jerivas**
Closes #56

image

### CODE REVIEW

File: generate_site/generate.py [hunk 1]
```diff
@@ -18,9 +19,16 @@
 df_org.fillna("", inplace=True)
 # TODO: Download images? 'Headshot image' is the column name
 
+# Assuming https://github.com/pyladies/global-conference/pull/60 gets merged
+with open("speakers.csv", encoding="utf-8-sig") as f:
```
Comment (jerivas) [inline generate_site/generate.py:23]: A new file `speakers.csv` has been created as part of this PR. Editing it and regenerating the site will update the Speakers section of the homepage. Keep in mind only the first 4 speakers are rendered in the homepage, but all speaker data can be used to implemented future features like #57

File: generate_site/generate.py [hunk 2]
```diff
@@ -18,9 +19,16 @@
...
+    reader = csv.DictReader(f)
+    df_speakers = list(reader)
+
 ##### Template configuration
 conf = {
     "ORG": df_org.to_dict("records"),
+    "SPEAKERS_ENABLED": False, # Toggle the "Speakers" section in the homepage
```
Comment (jerivas) [inline generate_site/generate.py:30]: Set this to `True` to see the new section

**Comment by Mariatta** [comment]
This is awesome, thank you!

**Comment by cmaureir** [comment]
Thanks @jerivas for the contribution :tada:

**Pull request merged**